src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorSingleStream.java

```diff
@@ -150,14 +173,9 @@ public long getTimeUntilNextRun()
      * {@inheritDoc}
      */
     @Override
-    public void incomingPacket(
-            long arrivalTimeMs,
-            int payloadSize,
-            int ssrc,
-            long rtpTimestamp,
-            boolean wasPaced)
+    public RawPacket reverseTransform(RawPacket pkt)
     {
-        Integer ssrc_ = ssrc;
+        Integer ssrc_ = pkt.getSSRC();
```

Comment:
if it isn't too much of a hassle, i think it'd be great to get rid of one more instance of this, but if you have to do a big refactoring nightmare to get that one changed over then i think we can punt on it.